docs/source/Installation/installation.rst

```diff
@@ -16,12 +16,16 @@ Once you are ready to install it in your own machine, go to the next section.
 Installing Solcore
 ^^^^^^^^^^^^^^^^^^
 
-Solcore is written mostly in Python, but the Poisson-Drift-diffusion (PDD) solver is written in Fortran to make it more efficient. The following instructions are expected to work in most systems, but check the sections below for OS specific instructions and caveats. In order to install Solcore in your computer, you will need the following:
+Solcore is written mostly in Python, but the Poisson-Drift-diffusion (PDD) solver is written in Fortran to make it more efficient,
+and the RCWA (rigorous coupled-wave analysis) solver uses the package S4 which is written in C++.
+The following instructions are expected to work in most systems, but check the sections below for OS specific instructions and caveats.
+In order to install Solcore in your computer, you will need the following:
 
 - Python >3.7
 - pip
 - setuptools
 - :doc:`a suitable Fortran compiler (only needed for the PDD solver) <compilation>`
+- :doc:`the S4 package <s4_installation>` (only needed for RCWA functionality)
```

Comment:
For consistency with the previos step. Obviously, you can change the Fortran-related step to match this one, instead.
```suggestion
- :doc:`the S4 package (only needed for RCWA functionality) <s4_installation>`
```